src/containers/BirdsBlock/BirdsBlock.tsx

```diff
@@ -0,0 +1,101 @@
+import { DescriptionBird } from '@view/DescriptionBird/DescriptionBird';
+
+import { ADD_POINTS } from '@constans/reducerActionType';
+import { useDispatch } from 'react-redux';
+import { IBird } from '@constans/interface/birds.interface';
+import { IPropsValueBirds } from './BirdsBlock.interface';
+
+import useSound from 'use-sound';
+import alarmRight from '@static/alarmValue/right.mp3';
+import alarmError from '@static/alarmValue/error.mp3';
+
+const BirdsBlock: React.FC<IPropsValueBirds> = ({ birds, url, setMainBird, setPercentage, percentage }) => {
+  const [bird, addBird] = useState<IBird[]>([]);
+  const [disabled, setDisabled] = useState(true);
+  const [styleBtn, setStyleBtn] = useState(`${styles.nexBtn}`);
+  const [points, setPoints] = useState(5);
+  const [audioControl, setAudioControl] = useState(0);
+
+  const [birdsArr, setBirdsArr] = useState<IBird[]>([]);
+
+  const dispatch = useDispatch();
+
+  const [playError] = useSound(alarmError);
+  const [playRight] = useSound(alarmRight);
+
+  const addPoints = (points: number) => {
+    dispatch({ type: ADD_POINTS, payload: points });
+  };
+
+  const onBirdNameClick = (bird: IBird) => {
+    setAudioControl(0);
+
+    if (bird) {
+      addBird([bird]);
+    }
+    if (!bird?.isRightAnswer) {
+      points ? setPoints(points - 1) : setPoints(0);
+      if (disabled) playError();
+    }
+    if (bird?.isRightAnswer) {
+      const [mainBird] = birds.filter((bird: IBird) => bird.isRightAnswer);
+      setMainBird(mainBird);
+      setDisabled(false);
+      setStyleBtn('nextBtnRight');
+      addPoints(points);
+      if (disabled) playRight();
+    }
+    if (disabled) {
+      setBirdsArr([...birdsArr, bird]);
+    }
+  };
+
+  const clearList = () => {
+    addBird([]);
+    setMainBird({
+      name: '******',
+      img: 'https://birds-quiz.netlify.app/static/media/bird.06a46938.jpg',
+    });
+
+    setStyleBtn('');
+    setDisabled(true);
+    setPoints(5);
+    setPercentage(0);
+    setBirdsArr([]);
+  };
+
+  return (
+    <div>
+      <div className={styles.ValueBlockBirds}>
+        <ul className={styles.valuesMenu}>
+          {birds.map((bird: IBird) => {
+            const stateStyleValue = birdsArr?.find((el: IBird) => el.id === bird.id);
+            return (
+              <Bird
+                key={bird.id}
+                onBirdNameClick={onBirdNameClick}
+                bird={bird}
+                designType={bird.isRightAnswer ? 'right' : 'error'}
+                stateStyleValue={!!stateStyleValue}
+              />
+            );
+          })}
+        </ul>
+        {bird.length === 0 || bird === undefined ? (
+          <DescriptionBirdDefault />
+        ) : (
+          <DescriptionBird birdValue={bird[0]} setPercentage={setAudioControl} percentage={audioControl} />
+        )}
+      </div>
+      <ButtonNavigation url={url} onClick={clearList} disabled={disabled} styleBtn={styleBtn} />
```

Comment:
styleBtn prop.

1) Не храни ее в useState styleBtn. Это просто бысмысленно. У тебя этот проп должен меняться в зависмости от того выбрана ли правильная птичка
вместо cosnt [styleBtn, setStyleBtn] = useState('')

const [isRightAnswerSelected, setRightAnswerSelected] = useState(false)

```suggestion
      <ButtonNavigation url={url} onClick={clearList} disabled={disabled} designType={isRightAnswerSelected ? 'primary' : undefined} />
```


2) styleBtn - очень плохое название. СтильКнопка. Ну зачем ? Я и так вижу что это кнопка. У кнопок есть designType, type. Вот подходящие название